### PR TITLE
Live Push 2021-07-03: fix visible slide range calculation

### PR DESCRIPTION
**Pull request by priscilamtzfraire**
### Code Refactoring

- Handle absolute indexes in the live region getter (3cfeb823d4ed21c79ce3fcc26fac77281cc4a32c)

---

### Backstory

Trying to solve [this issue](https://github.com/justia/justatic/issues/2441), I noticed the `tns-slide-active` class wasn't being applied correctly; instead of adding it to the active slides, it was added to the previous slides. This was because the [`updateSlideStatus`](https://github.com/justia/tiny-slider/blob/master/src/tiny-slider.js#L1929) function was getting the wrong `start` and `end` values (they come from the `getVisibleSlideRange` function, which I modified in PR #1).

Long story short: I reverted the fix I just mentioned and isolated that solution to the [`getLiveRegionStr`](https://github.com/justia/tiny-slider/blob/master/src/tiny-slider.js#L1713) function only.

### CODE REVIEW

File: src/tiny-slider.js [hunk 1]
```diff
@@ -1712,9 +1712,9 @@ export var tns = function(options) {
 
   function getLiveRegionStr () {
     var arr = getVisibleSlideRange(),
-        start = arr[0] + 1,
-        end = arr[1] + 1;
-    return start === end ? arr[0] : arr[0] + ' to ' + arr[1];
+        start = getAbsIndex(arr[0]) + 1,
+        end = getAbsIndex(arr[1]) + 1;
+    return start === end ? start : start + ' to ' + end;
```
Comment (Digznav) [inline src/tiny-slider.js:1717]: Please change it to template literals.

**Review by Digznav — APPROVED** [review]

**Review by Digznav — APPROVED** [review]
